Ccq/etc/queue_publisher.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/publisher.xsd">
+    <publisher topic="rltsquare">
+        <connection name="amqp" exchange="rltsquare"/>
```

Comment:
Overcomplicated code - there is no need to declare connection here, the default will be picked up.